python_fundamentals-master/18_lambdas/18_04_hello.py

```diff
@@ -4,3 +4,6 @@
 What does it return?
 
 '''
+
+get_hello = lambda: "Hello world"
+print(get_hello())
```

Comment:
Technically you were supposed to make it print hello world within the lambda

```python
get_hello = lambda: print("hello world")
a = get_hello()
type(a)
```
and see what a is.  It will be `None` because when you print something I guess lambda exit? Not sure why.